docs/user-guide/configuring.md

```diff
@@ -344,7 +344,7 @@ The second way to use configuration files is via `.eslintrc` and `package.json`
 
 In each case, the settings in the configuration file override default settings.
 
-## Configuration Cascading and Hierarchy
+## Configuration Cascading
```

Comment:
The problem with splitting them is that they aren't actually two different things. Cascading is a function of the hierarchy.